[Jenkins Keycloak CI] - Cookies Tests - KcOidcBrokerPrivateKeyJwtCustomAudienceTest

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

```
[ERROR] org.keycloak.testsuite.broker.KcOidcBrokerPrivateKeyJwtCustomAudienceTest.loginWithExistingUser  Time elapsed: 143.118 s  <<< ERROR!
org.openqa.selenium.TimeoutException: 
Expected condition failed: waiting for value to contain (ignoring case) "update account information". Current value: "We are sorry..." (tried for 5 second(s) with 500 milliseconds interval)
Build info: version: '4.25.0', revision: '8a8aea2337'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.11.6-200.fc40.x86_64', java.version: '17.0.13'
Driver info: org.openqa.selenium.firefox.FirefoxDriver_ByGraphene
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 132.0, moz:accessibilityChecks: false, moz:buildID: 20241025123907, moz:debuggerAddress: 127.0.0.1:10712, moz:geckodriverVersion: 0.35.0, moz:headless: false, moz:platformVersion: 6.11.6-200.fc40.x86_64, moz:processID: 145457, moz:profile: /tmp/rust_mozprofilepsSp2k, moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:cdp: ws://127.0.0.1:10712/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, userAgent: Mozilla/5.0 (X11; Linux x86...}
Session ID: bf4cbf10-4e9c-41d6-8ef4-931b6426d33d
        at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
        at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
        at org.keycloak.testsuite.broker.BrokerTestTools.waitForPage(BrokerTestTools.java:89)
        at org.keycloak.testsuite.broker.AbstractBrokerTest.loginUser(AbstractBrokerTest.java:70)
        at org.keycloak.testsuite.broker.AbstractBrokerTest.testLogInAsUserInIDP(AbstractBrokerTest.java:59)
        at org.keycloak.testsuite.broker.AbstractBrokerTest.loginWithExistingUser(AbstractBrokerTest.java:147)

```

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

`mvn -f testsuite/integration-arquillian/tests/base/pom.xml clean install -Pauth-server-quarkus -Pfirefox-strict-cookies -Dauth.server.host=auth-server-127-0-0-1.nip.io -Dauth.server.host2=127.0.0.1.sslip.io -Dtest=KcOidcBrokerPrivateKeyJwtCustomAudienceTest`

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcOidcBrokerPrivateKeyJwtCustomAudienceTest.java
+++ b/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcOidcBrokerPrivateKeyJwtCustomAudienceTest.java
@@ -32,6 +32,7 @@ import java.util.Map;
 import static org.keycloak.testsuite.broker.BrokerTestConstants.IDP_OIDC_ALIAS;
 import static org.keycloak.testsuite.broker.BrokerTestConstants.IDP_OIDC_PROVIDER_ID;
 import static org.keycloak.testsuite.broker.BrokerTestTools.createIdentityProvider;
+import static org.keycloak.testsuite.util.ServerURLs.AUTH_SERVER_HOST2;
 
 public class KcOidcBrokerPrivateKeyJwtCustomAudienceTest extends AbstractBrokerTest {
 
@@ -64,7 +65,7 @@ public class KcOidcBrokerPrivateKeyJwtCustomAudienceTest extends AbstractBrokerT
             applyDefaultConfiguration(config, syncMode);
             config.put("clientSecret", null);
             config.put("clientAuthMethod", OIDCLoginProtocol.PRIVATE_KEY_JWT);
-            config.put("clientAssertionAudience", "https://localhost:8543/auth/realms/provider");
+            config.put("clientAssertionAudience", "https://" + AUTH_SERVER_HOST2 + ":8543/auth/realms/provider");
             return idp;
         }
 

